src/main/java/entity/ForumEntry.java

```diff
@@ -0,0 +1,80 @@
+package entity;
+
+import java.time.LocalDateTime; 
+
+public abstract class ForumEntry {
+    private String entryID;
+    private User author;
+    private Content content;
+    private LocalDateTime postedDate;
+    private LocalDateTime editDate; 
```

Comment:
Would lastModifiedDate be more descriptive?
```suggestion
    private LocalDateTime lastModifiedDate; 
```